### PR TITLE
Fix bottom space from DayTab offsets in Timetable Screen

### LINKED ISSUE #914: In TimetableList, Items cannot be displayed at the bottom of the screen by the offset of DayTab
## Overview
As the title says, the DayTab offset moves when scrolling, making it impossible to display the contents of the list at the bottom.

https://github.com/user-attachments/assets/1acdb398-aeae-4f0b-99ff-3ceb69a838b8

### PR DESCRIPTION
**Pull request by kktaro**
## Issue
- close #914

## Overview (Required)
- In Timetable Screen,  remove bottom empty space due to `DateTab` offset.
- Replace **Column and offset** with **Box and Padding**.

## Links
- 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 | 

### portrait
Before | After
:--: | :--:
<video src="https://github.com/user-attachments/assets/547da23c-8def-4563-b867-ce461712bf69" width="300" > | <video src="https://github.com/user-attachments/assets/fd22ca7d-d5f2-4fae-aeab-51cc43bc3fe5" width="300" >

### landscape

Before | After
:--: | :--:
<video src="https://github.com/user-attachments/assets/2aaf0623-45bc-4d16-9441-f2e57d36838b" width="300" > | <video src="https://github.com/user-attachments/assets/b3151287-37b3-4153-ab2d-8d14d1b1e512" width="300" >

### CODE REVIEW

**Comment by takahirom** [comment]
Could you check the grid behavior?

**Comment by kktaro** [comment]
Sorry.
In TimetableGrid Screen, modifier from arguments is changed to apply to the whole screen.

image

File: feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/section/TimetableGrid.kt [hunk 1]
```diff
@@ -139,7 +139,7 @@ fun TimetableGrid(
     val animatedScope = LocalAnimatedVisibilityScope.current
 
     Row(
-        modifier = Modifier
```
Comment (takahirom) [inline feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/section/TimetableGrid.kt:142]: Thanks!

**Review by takahirom — APPROVED** [review]
This is important fix! Thank you for your contribution!

**Pull request merged**